packages/bitcoin/src/utils/bitcoin.utils.ts

```diff
@@ -100,7 +100,10 @@ export function decodeBitcoinTx(tx: string): ReturnType<typeof btc.RawTx.decode>
   return btc.RawTx.decode(hexToBytes(tx));
 }
 
-export function getAddressFromOutScript(script: Uint8Array, bitcoinNetwork: BtcSignerNetwork) {
+export function getAddressFromOutScript(
+  script: Uint8Array,
+  bitcoinNetwork: BtcSignerNetwork
+): BitcoinAddress | string {
```

Comment:
This function sometimes returns an empty string / unknown so it needs to be `BitcoinAddress | string`. Maybe that's something we need to fix?

```
case "unknown":
      return "unknown";
    case "tr_ms":
    case "tr_ns":
    default:
      return "";
```